infra/helm/meshforms/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: meshforms
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
```

Comment:
I think you wanted all this removed in meshdb

Reply:
Actually, these variables are necessary and we should keep them. I think they have something to do with stuff like `.Chart.Name` working.

Reply:
Comments

Reply:
oh god I need to go to bed.